src/app/generic-form/generic-form.module.ts

```diff
@@ -36,7 +40,15 @@ import { GenericFormComponent } from './generic-form/generic-form.component';
     BrowserModule,
     BrowserAnimationsModule,
     MaterialDesignFrameworkModule,
-    JsonSchemaFormModule.forRoot(MaterialDesignFrameworkModule),
+    {
```

Comment:
Why is this expanded now?

Reply:
https://github.com/dschnelldavis/angular2-json-schema-form/issues/189